egs/librispeech/ASR/pruned_transducer_stateless2/conformer.py

```diff
@@ -1098,7 +1098,10 @@ def multi_head_attention_forward(
             head_dim * num_heads == embed_dim
         ), "embed_dim must be divisible by num_heads"
 
-        scaling = float(head_dim) ** -0.5
+        scaling = (
```

Comment:
Can we register it as a buffer in the constructor?

Reply:
Actually I just closed laptop on friday, and opened again on monday (today), then I am not facing with this problem anymore. Same environment, same device, same codes. torch.onnx.export() function exports a graph which doesn't contain double type tensor. There are same issues: [1](https://github.com/microsoft/CodeXGLUE/issues/65), [2](https://github.com/pytorch/pytorch/issues/58367), [3](https://github.com/microsoft/onnxruntime/issues/1764). I also create a new environment to re-produce, but I couldn't reproduce, everything works well. Still, I can register it as a buffer in the constructor and make a commit, someone may face with this in the future?

Reply:
In that case, I suggest leaving the current code as it is.

We can fix it if someone can reproduce it.